### ISSUE TITLE
Molniya example doesn't plot in Jupyter/Python3 on Mac or Fedora

### ISSUE THREAD

**Issue by MattEttus** [poliastro/poliastro#290]
I copied the 3 lines of code exactly and they don't produce a plot, just a 2-element list of matplotlib objects.  Possibly it is a problem with matplotlib, I am using conda-forge

🖥 **Please paste the output of following commands**

`conda info -a`

```
Current conda install:

               platform : osx-64
          conda version : 4.3.30
       conda is private : False
      conda-env version : 4.3.30
    conda-build version : 3.0.27
         python version : 3.6.3.final.0
       requests version : 2.18.4
       root environment : /Users/mettus/anaconda3  (writable)
    default environment : /Users/mettus/anaconda3
       envs directories : /Users/mettus/anaconda3/envs
                          /Users/mettus/.conda/envs
          package cache : /Users/mettus/anaconda3/pkgs
                          /Users/mettus/.conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/main/osx-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/osx-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /Users/mettus/.condarc
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.30 requests/2.18.4 CPython/3.6.3 Darwin/17.2.0 OSX/10.13.1    
                UID:GID : 501:20
# conda environments:
#
ds                       /Users/mettus/anaconda3/envs/ds
ds.bak                   /Users/mettus/anaconda3/envs/ds.bak
root                  *  /Users/mettus/anaconda3

sys.version: 3.6.3 |Anaconda custom (64-bit)| (defaul...
sys.prefix: /Users/mettus/anaconda3
sys.executable: /Users/mettus/anaconda3/bin/python
conda location: /Users/mettus/anaconda3/lib/python3.6/site-packages/conda
conda-build: /Users/mettus/anaconda3/bin/conda-build
conda-convert: /Users/mettus/anaconda3/bin/conda-convert
conda-develop: /Users/mettus/anaconda3/bin/conda-develop
conda-env: /Users/mettus/anaconda3/bin/conda-env
conda-index: /Users/mettus/anaconda3/bin/conda-index
conda-inspect: /Users/mettus/anaconda3/bin/conda-inspect
conda-metapackage: /Users/mettus/anaconda3/bin/conda-metapackage
conda-render: /Users/mettus/anaconda3/bin/conda-render
conda-server: /Users/mettus/anaconda3/bin/conda-server
conda-skeleton: /Users/mettus/anaconda3/bin/conda-skeleton
conda-verify: /Users/mettus/anaconda3/bin/conda-verify
user site dirs: ~/.local/lib/python3.6

CIO_TEST: <not set>
CONDA_DEFAULT_ENV: <not set>
CONDA_ENVS_PATH: <not set>
CONDA_ROOT: /Users/mettus/anaconda3
DYLD_LIBRARY_PATH: <not set>
FTP_PROXY: <not set>
HTTPS_PROXY: <not set>
HTTP_PROXY: <not set>
LD_LIBRARY_PATH: <not set>
PATH: /Users/mettus/anaconda3/bin:/Users/mettus/.bin:/Users/mettus/.rvm/gems/ruby-2.2.2/bin:/Users/mettus/.rvm/gems/ruby-2.2.2@global/bin:/Users/mettus/.rvm/rubies/ruby-2.2.2/bin:/opt/local/bin:/opt/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Users/mettus/.rvm/bin
PYTHONHOME: <not set>
PYTHONPATH: <not set>
REQUESTS_CA_BUNDLE: <not set>
SSL_CERT_FILE: <not set>

WARNING: could not import _license.show_info
# try:
# $ conda install -n root _license

`python -c "import poliastro.testing; poliastro.testing.test()"`

$ python -c "import poliastro.testing; poliastro.testing.test()"
===================================================== test session starts =====================================================
platform darwin -- Python 3.6.3, pytest-3.2.1, py-1.4.34, pluggy-0.4.0
rootdir: /Users/mettus, inifile:
collected 138 items                                                                                                            

../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_bodies.py ......
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_coordinates.py ...
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_ephem.py .
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_hyper.py ...........
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_iod.py .........
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_jit.py ...
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_maneuver.py ......
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_patched_conics.py ..
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_plotting.py ......
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_plotting3d.py ......
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_stumpff.py ...
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_twobody.py ..
../anaconda3/lib/python3.6/site-packages/poliastro/tests/test_util.py .......
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_neos/test_dastcom5.py .........
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_neos/test_neos_neows.py ......
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_twobody/test_angles.py .......
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_twobody/test_decorators.py ..
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_twobody/test_orbit.py ..................
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_twobody/test_propagation.py .........
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_twobody/test_sample.py .xx.........
../anaconda3/lib/python3.6/site-packages/poliastro/tests/tests_twobody/test_states.py ..........

============================================ 136 passed, 2 xfailed in 6.49 seconds ============================================
```

🎯 **Goal**

I can't use the package without it :)

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
Hi @MattEttus, thanks for taking the time of reporting an issue! I edited the formatting a little bit.

You are right that the documentation is incomplete, and this is in fact the code that is needed to have a plot:

```
import matplotlib.pyplot as plt
plt.ion()

from poliastro.examples import molniya
from poliastro.plotting import plot

plot(molniya)
```

Could you please try this and see if it works? Also, would you like to make the change yourself? It would be very appreciated :smiley: You can use the GitHub interface:

https://github.com/poliastro/poliastro/edit/master/docs/source/index.rst

**Comment by MattEttus** [comment]
Yes, that works.  Thanks!  I created a pull request to include this in the docs.

**Comment by Juanlu001** [comment]
Closed in #291.
